cogs/highlights.py

```diff
@@ -69,11 +69,21 @@ async def _send_highlights(
                     continue
         await con.commit()
 
+    def _get_guild_id(self, server: discord.Guild | None) -> int:
+        return server.id if server else 0
+
+    def _server_text(self, server, pre="", no_server="globally"):
```

Comment:
This should be a function and the names should be clarified for better readability (`pre` -> `prefix`). `if server else no_server` should not be part of the function but in-lined where necessary. It also needs type hints.

Reply:
Fixed: simplified (optional args removed), moved to function "_for_guild_str(guild)".